src/api/test/models/status_history_test.rb

```diff
@@ -25,4 +29,20 @@ def teardown
     assert_equal 1, rel.count
     assert_equal 499.5, rel.first.value
   end
+
+  test "history_by_key_and_hours" do
+    Timecop.freeze(2010, 7, 12)
+    day_before_yesterday = Time.now.to_i - 2.days
+    yesterday = Time.now.to_i - 1.days
+    StatusHistory.transaction do
+      10.times do |i|
+        StatusHistory.create time: day_before_yesterday + i, key: 'squeue_low_aarch64', value: i
+      end
+      5.times do |i|
+        StatusHistory.create time: yesterday + i, key: 'squeue_low_aarch64', value: i
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.